src/Tizen.Network.Stc/Tizen.Network.Stc/StcRule.cs

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// A class for managing the Stc Rules to match applications.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class StcRule
```

Comment:
Let's think about naming of `StcRule`
Why it is StcRule?
It is a class for represent a filter to use on retrieval of Statistics data. 
So, I think, StatisticsFilter is better name.
a case of PackageManager
https://github.com/Samsung/TizenFX/blob/f7c1f3b65cd7a6f8a5f86899750f5d49c7ccda64/src/Tizen.Applications.PackageManager/Tizen.Applications/PackageFilter.cs#L25

a case of ApplicationInfo
https://github.com/Samsung/TizenFX/blob/f7c1f3b65cd7a6f8a5f86899750f5d49c7ccda64/src/Tizen.Applications.Common/Tizen.Applications/ApplicationInfoFilter.cs#L26

Calendar
https://github.com/Samsung/TizenFX/blob/27893e385679cbc0a0421355866043e415e55977/src/Tizen.Pims.Calendar/Tizen.Pims.Calendar/CalendarFilter.cs#L26

Reply:
Changed as per suggestion.